cmd/indexer.go

```diff
@@ -71,7 +75,12 @@ func startIndexer(*cobra.Command, []string) {
 		"discover": len(indexerArgs.nodes.DiscoveryNode) > 0,
 	}).Info("Starting indexer service ...")
 
-	util.MustServeRPC(indexerArgs.endpoint, map[string]interface{}{
-		api.Namespace: api,
+	gateway.MustServeWithRPC(gateway.Config{
+		Endpoint:            indexerArgs.endpoint,
+		Nodes:               indexerArgs.nodes.TrustedNodes,
+		MaxDownloadFileSize: indexerArgs.maxDownloadFileSize,
+		RPCHandler: util.MustNewRPCHandler(map[string]interface{}{
```

Comment:
here means that users can still use cli to download file with rpc instead of http request, am I understanding correctly?

Reply:
CLI is the same as before, just supports to download files via REST API. This is helpful for web page to download files instead of CLI.